instrgen/README.md

```diff
@@ -0,0 +1,25 @@
+# OpenTelemetry Go Source Automatic Instrumentation
+
+This repository provides [OpenTelemetry] source instrumentation for [Go]
+by automatically modifying source code in similar way as compiler.
+
+## Project Status
+
+:construction: This project is currently work in progress.
+
+### Compatibility
+
+Tool is based on Go standard library and is platform agnostic.
+
+## Contributing
+
+See the [contributing documentation](./CONTRIBUTING.md).
+
+## License
+
+OpenTelemetry Go Source Automatic Instrumentation is licensed under the terms of the [Apache Software License version 2.0].
+See the [license file](./LICENSE) for more details.
+
+[OpenTelemetry]: https://opentelemetry.io/
+[Go]: https://go.dev/
+[Apache Software License version 2.0]: https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
These are part of the repository root and not needed at the package level.

```suggestion
[OpenTelemetry]: https://opentelemetry.io/
```

Reply:
updated